velox/connectors/Connector.h

```diff
@@ -136,6 +136,12 @@ class DataSink {
   /// TODO maybe at some point we want to make it async.
   virtual void appendData(RowVectorPtr input) = 0;
 
+  /// Returns the number of bytes written on disk by this data sink so far.
+  /// If size is not available, this method should return zero.
```

Comment:
Drop "/// If size is not available, this method should return zero."? We must support this? And put TODO in derived DataSink class implementation if not supported yet. Thanks!